### PR TITLE
[billing] GetLatestInvoice RPC protos

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Adds RPC definitions

Part of https://github.com/gitpod-io/gitpod/issues/11692

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: components/usage-api/usage/v1/billing.proto [hunk 1]
```diff
@@ -23,3 +26,14 @@ message UpdateInvoicesRequest {
...
+message GetLatestInvoiceRequest {
+  string team_id = 1;
+  string user_id = 2;
+}
+
+message GetLatestInvoiceResponse {
+  string invoice_id = 1;
+  string currency = 2;
+  double amount = 3;
+}
```
Comment (andrew-farries) [inline components/usage-api/usage/v1/billing.proto:39]: Do we also want to send the `team_id` or `user_id` back as part of the response too?

File: components/usage-api/usage/v1/billing.proto [hunk 2]
```diff
@@ -23,3 +26,14 @@ message UpdateInvoicesRequest {
 
 message UpdateInvoicesResponse {
 }
+
+message GetLatestInvoiceRequest {
+  string team_id = 1;
+  string user_id = 2;
```
Comment (andrew-farries) [inline components/usage-api/usage/v1/billing.proto:32]: Should this be a [oneof](https://developers.google.com/protocol-buffers/docs/proto3#oneof) field?

Comment (easyCZ) [reply components/usage-api/usage/v1/billing.proto]: updated to one of

File: components/usage-api/usage/v1/billing.proto (hunk 1 above)
Comment (easyCZ) [reply components/usage-api/usage/v1/billing.proto:41]: I'd leave it out for now. The caller knows the user_id or team_id so I think for this case they can re-use that.

**Review by andrew-farries — APPROVED** [review]

**Pull request merged**